src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/projectcreation/view/fragments/SelectChildren.kt

```diff
@@ -0,0 +1,91 @@
+package org.wycliffeassociates.otter.jvm.app.ui.projectcreation.view.fragments
+
+import de.jensd.fx.glyphs.materialicons.MaterialIcon
+import de.jensd.fx.glyphs.materialicons.MaterialIconView
+import javafx.geometry.Insets
+import javafx.geometry.Pos
+import javafx.scene.Node
+import javafx.scene.control.ContentDisplay
+import javafx.scene.control.Label
+import org.wycliffeassociates.otter.jvm.app.ui.imageLoader
+import org.wycliffeassociates.otter.jvm.app.ui.projectcreation.SlugsEnum
+import org.wycliffeassociates.otter.jvm.app.ui.projectcreation.viewmodel.ProjectCreationViewModel
+import org.wycliffeassociates.otter.jvm.app.ui.styles.AppStyles
+import org.wycliffeassociates.otter.jvm.app.ui.styles.ProjectWizardStyles
+import org.wycliffeassociates.otter.jvm.app.widgets.wizardcard
+import tornadofx.*
+import java.io.File
+
+class SelectChildren : View() {
```

Comment:
SelectCollection might be better. We're selecting children for a collection from the previous screen, but on this screen we're just selecting from a list of collections